site/content/docs/dashboards/_index.md

```diff
@@ -1,12 +1,13 @@
 ---
-title: Url and Custom Domain
+title: Overview
```

Comment:
This page should show at least an example screenshot of a complete dashboard. We are talking about dashboards in this section without ever showing what they look like.